src/app/vendors/VendorItemComponent.tsx

```diff
@@ -22,11 +22,13 @@ export default function VendorItemComponent({
   defs,
   owned,
   characterId,
+  hideCosts,
 }: {
   defs: D2ManifestDefinitions;
   item: VendorItem;
   owned: boolean;
   characterId?: string;
+  hideCosts?: boolean;
```

Comment:
If you're hiding the costs, maybe don't use VendorItemComponent at all?

Reply:
+1 am just using a `<ItemPopupTrigger><ConnectedInventoryItem /></>` directly